src/main/java/school/lemon/changerequest/java/FractionNumberDemo.java

```diff
@@ -0,0 +1,26 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 14.12.16.
+ */
+public class FractionNumberDemo {
+    public static void main(String[] args) {
```

Comment:
You should add demos for all possible operations and cover all requirements of task.
Here you don't have demo to show that creation of FractionNumber with 0 divisor is prohibited.